dotnet/src/SemanticKernel.Abstractions/SemanticKernel.Abstractions.csproj

```diff
@@ -18,10 +18,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
-    <PackageReference Include="Microsoft.Bcl.HashCode" />
```

Comment:
Should this be in the `netstandard2.0` item group?

Reply:
U are right will make that additional cleanup tomorrow morning my time.

Reply:
@markwallace-microsoft  it appears that Microsoft.BCL.HashCode is not needed at all & Microsoft.Bcl.AsyncInterfaces is being imported as a transitive dependency. Not sure if the abstractions explicitly need this dependency.